### ISSUE TITLE
'No Match for argument: fakesystemd' if use centos 7.1 image

### ISSUE THREAD

**Issue by bear0330** [CentOS/sig-cloud-instance-images#20]
The documentation on https://registry.hub.docker.com/_/centos/ says:
Dockerfile for systemd base image:

```
....
RUN yum -y swap -- remove fakesystemd -- install systemd systemd-libs
....
```

But this line will cause problem: `No Match for argument: fakesystemd` if we use latest centos image.
systemd won't be installed.

**Comment by jperrin** [comment]
Correct. I'm working on updating the documentation now to push to docker, as in the current image we ship systemd-container instead of fakesystemd. 

There is currently an issue with this ( https://github.com/CentOS/sig-cloud-instance-images/issues/19 ) I was trying to resolve before committing the documentation update.

**Comment by bear0330** [comment]
Hello, I saw you committed some code for #19. How about this issue? I need to rebuild my base images to try anything else, but I don't know what is right way to do, so I am waiting for your official documentation to follow. I very appreciate your help. Thanks.

**Comment by jperrin** [comment]
what issue? the information was provided in my comment. fakesystemd is no longer present in the 7 and latest tags.. There's a package called systemd-container included now that makes systemd work in containers. you simply need to mount /sys/fs/cgroups in the container, and follow the comments in the Dockerfile -> https://github.com/CentOS/sig-cloud-instance-images/blob/CentOS-7/docker/Dockerfile

The documentation should be up soon on the actual docker index page.

**Comment by kjenney** [comment]
I'm unclear why the documentation is STILL outdated months later. If you expect people to find documentation within the origin dockerfile then remove the inaccurate information on the image documentation page.

**Comment by jperrin** [comment]
There were two pull requests submitted to update the documentation that were not accepted for various reasons.. In the upcoming version (7.2.1511) The directions will change again. Once that release is done, I'll do another PR to update to the documentation and we'll go from there.
